web/js/components/util/about-page.js

```diff
@@ -0,0 +1,322 @@
+import React from 'react';
+import Scrollbar from './scrollbar';
+
+export default function AboutPage() {
+  return (
+
+    <Scrollbar style={{ height: 'calc(100vh - 200px)' }}>
+      <div id="page" className="wv-document">
+        <div className="right">
+          <div className="caption">
+            Version @BUILD_VERSION@
+            (
+            <a href="https://github.com/nasa-gibs/worldview/releases" target="_blank" rel="noopener noreferrer">Release Notes</a>
+            )
+          </div>
+        </div>
+
+        <h1>
+          Welcome to @NAME@
+        </h1>
+
+        <p>
+          This app from NASA&apos;s
+          {' '}
+          <a href="https://earthdata.nasa.gov/eosdis" target="_blank" rel="noopener noreferrer">EOSDIS</a>
+          {' '}
+          provides the capability to interactively browse over 900 global, full-resolution satellite imagery
+          layers and then download the underlying data. Many of the imagery layers are updated daily and are available within three hours
+          of observation - essentially showing the entire Earth as it looks &quot;right now&quot;. This supports time-critical application
+          areas such as wildfire management, air quality measurements, and flood monitoring. Arctic and Antarctic views
+          of many products are also available for a &quot;full globe&quot; perspective. Geostationary imagery layers are also now available.
+          These are provided in ten minute increments for the last 30 days. These full disk hemispheric views allow for almost real-time viewing of
+          changes occurring around most of the world. Browsing on tablet and smartphone devices is generally supported for mobile access to the imagery.
+        </p>
+
+        <div className="right">
+          <a href="https://earthdata.nasa.gov/gibs" className="gibs-logo" target="_blank" rel="noopener noreferrer" />
```

Comment:
This GIBS logo doesn't appear to be showing up when I test this locally.
<img width="620" alt="Screen Shot 2020-04-24 at 1 44 17 PM" src="https://user-images.githubusercontent.com/1480000/80241642-f16a0c80-8631-11ea-9ea0-1e28b35fc7ae.png">